### ISSUE TITLE
Look at Bayes Non parametric learning

### ISSUE THREAD

**Issue by MichaelClerx** [pints-team/pints#850]
"Bayesian posterior bootstrap"

Lyndon, Holmes, walker 2018 I think

Looks cool

**Comment by MichaelClerx** [comment]
Intended for when the model is wrong. Should be good for multimodal distributions

**Comment by MichaelClerx** [comment]
https://papers.nips.cc/paper/7477-nonparametric-learning-from-bayesian-models-with-randomized-objective-functions.pdf

**Comment by chonlei** [comment]
I would like to look into this too. And here is another paper that the above paper extends from:

https://academic.oup.com/biomet/article/106/2/465/5385582

**Comment by MichaelClerx** [comment]
Cool! We should also remember to check when Chris Holmes talk goes online on the Newton website. Would be good to save a link :-)

**Comment by chonlei** [comment]
> Cool! We should also remember to check when Chris Holmes talk goes online on the Newton website. Would be good to save a link :-)

https://www.newton.ac.uk/seminar/20190605140015001

**Comment by MichaelClerx** [comment]
Seems to require multiple optimisation problems to be solved every iteration??

**Comment by mirams** [comment]
One optimisation problem per 'sample' (like normal bootstrap) if I understood it correctly...

**Comment by chonlei** [comment]
Yes, "One optimisation problem per 'sample'", but if we want N=5000 samples, then we have to run 5000 times of optimisation -- which sounds heavily to me. But I remember Chris once said this method does not lose computational time compared to normal MCMC, which I might get it wrong but I don't quite understand that.
